redwood-widget/src/jsMain/kotlin/app/cash/redwood/widget/RedwoodHTMLElementView.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.widget
+
+import app.cash.redwood.ui.Margin
+import app.cash.redwood.ui.Size
+import app.cash.redwood.ui.UiConfiguration
+import app.cash.redwood.ui.dp
+import app.cash.redwood.widget.Widget.Children
+import kotlinx.browser.window
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import org.w3c.dom.HTMLElement
+import org.w3c.dom.MediaQueryList
+
+public fun HTMLElement.asRedwoodView(): RedwoodView<HTMLElement> {
+  checkNotNull(parentNode) {
+    "Element $this must be attached to Document to be used as RedwoodView"
+  }
+  return RedwoodHTMLElementView(this)
+}
+
+private class RedwoodHTMLElementView(
+  element: HTMLElement,
+) : RedwoodView<HTMLElement> {
+  override val children: Children<HTMLElement> = HTMLElementChildren(element)
+  override val uiConfiguration = MutableStateFlow(
+    UiConfiguration(
+      darkMode = window.matchMedia("(prefers-color-scheme: dark)").matches,
+      safeAreaInsets = Margin.Zero,
+      viewportSize = Size(width = element.offsetWidth.dp, height = element.offsetHeight.dp),
+      density = 1.0,
+    )
```

Comment:
Remove explicit setting of defaults?

```suggestion
    UiConfiguration(
      darkMode = window.matchMedia("(prefers-color-scheme: dark)").matches,
      viewportSize = Size(width = element.offsetWidth.dp, height = element.offsetHeight.dp),
    )
```

Reply:
Yep, done.